packages/client/src/components/SafeSettings/components/Owners/AddSafeOwner.js

```diff
@@ -49,22 +42,12 @@ const AddSafeOwner = ({ treasury, safeOwners }) => {
           <label className="has-text-grey mb-2 with-icon is-flex is-align-items-center">
             Address<span className="has-text-red">*</span> <Svg name="QuestionMark" className="ml-1" />
           </label>
-          <div style={{ position: "relative" }}>
-            <input
-              className={addrInputClasses}
-              type="text"
-              placeholder=""
-              value={address}
-              onChange={(e) => onAddressChange(e.target.value)}
-            />
-            {addressValid && (
-              <div
-                style={{ position: "absolute", right: 17, top: 19 }}
-              >
-                <Svg name="Check" height="15" width="15" />
-              </div>
-            )}
-          </div>
+          <InputAddress
+            className={addressValidClass}
```

Comment:
It looks like this addressValidClass depends if the address is valid, so that logic can be moved to InputAddress?
This way we can ensure all the inputs have is-success/is-error class attached

Reply:
@KsenijaZivkovic good call - added!